code/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,79 @@
+
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import frc.robot.common.IIntakeSubsystem;
+import frc.robot.common.ILogger;
+import frc.robot.common.TraceableSubsystem;
+
+public class Intake extends TraceableSubsystem implements IIntakeSubsystem {
+
+    public SpeedController intakeController = new PWMVictorSPX(1);
```

Comment:
I'm seeing a lot of magic numbers. We really need to get in the habit of adding these to the config class.